internal/pod_helper.go

```diff
@@ -226,20 +226,6 @@ func GetSidecarImage(cluster *fdbv1beta2.FoundationDBCluster, pClass fdbv1beta2.
 	return GetImage(image, imageConfigs, cluster.Spec.Version, false)
 }
 
-// CreatePodMap creates a map with the process group ID as a key and the according Pod as a value
-func CreatePodMap(cluster *fdbv1beta2.FoundationDBCluster, pods []*corev1.Pod) map[fdbv1beta2.ProcessGroupID]*corev1.Pod {
```

Comment:
This map needed to be built by all controllers/reconcilers (that is, all controllers that needed to use this map)? Why not build it once (and update the map where needed)?